src/bin/spin.rs

```diff
@@ -44,8 +44,12 @@ enum SpinApp {
     Bindle(BindleCommands),
     Deploy(DeployCommand),
     Build(BuildCommand),
+    #[clap(subcommand)]
+    Plugin(PluginCommands),
     #[clap(subcommand, hide = true)]
     Trigger(TriggerCommands),
+    #[clap(external_subcommand)]
```

Comment:
Is there a way to surface these to the help system?  When I run `cargo`, it lists all known commands including those from plugins - is it possible for us to do the same?  (Probably future enhancement, don't think it was discussed in the SIP.)

Reply:
I think cargo lists it when using the argument `--list`. It is a command that can be potentially added later or if we want the default `help` to display all the known commands including plugins, it can be accomplished by modifying the default help text.

Reply:
I stand corrected!  I saw `add` in the `cargo` output, but forgot that `add` no longer came from a plugin!  My apologies.  And agree later in a distinct command is fine.